logisland-docker/base/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM centos:6
+MAINTAINER Hurence
+
+
+USER root
+
+# install dev tools
+RUN yum clean all; \
+    rpm --rebuilddb; \
+    yum install -y curl which tar sudo openssh-server openssh-clients rsync wget git svn telnet nano vim;
+RUN yum groupinstall -y 'Development Tools';
+
+
+# Java 8
+RUN wget --no-cookies --no-check-certificate --header "Cookie: gpw_e24=http%3A%2F%2Fwww.oracle.com%2F; oraclelicense=accept-securebackup-cookie" "http://download.oracle.com/otn-pub/java/jdk/8u131-b11/d54c1d3a095b4ff2b6607d096fa80163/jre-8u131-linux-x64.rpm" && yum -y localinstall jre-8u131-linux-x64.rpm && rm jre-8u131-linux-x64.rpm
```

Comment:
Has you provide --no-cookies, there is no need to provide a cookie at all. you can execute the command as follows :
   wget --no-check-certificate --no-cookies --header "Cookie: oraclelicense=accept-securebackup-cookie"